# Easy come, easier go: mapping the loss of bacterial motility across the tree of life

Most bacterial swimming is powered by the bacterial flagellar motor, a nanomachine that self-assembles from up to 45 proteins into a membrane-spanning complex. The number and types of proteins involved in the flagellar motor vary widely. Predicting flagellar motility from genomic data can facilitate large-scale genomic studies where experimental validation may not be feasible. Using sequence and structural homology, we conducted a homology search for 54 flagellar pathway genes across 11,365 bacterial genomes. We developed and validated a classifier to predict whether a specific genome was motile and mapped the evolution of motility across the microbial tree of life. We determined that the ancestral state was motile, and the rate of loss of motility was 4 times the rate of gain.

Schematic overview of the flagellar protein-encoding gene homologue search in bacterial genomes. The query proteins assembled from six bacterial species were searched against the PATRIC database. The sequence homologues were further processed for downstream analysis, such as length filtering and structural similarity checks, to improve the confidence of the homology search. 

![Figure1_5x copy](https://github.com/user-attachments/assets/49edf13b-ce4b-4006-afd9-299d1d95378b)

